app/scripts/actions/index.js

```diff
@@ -264,6 +297,71 @@ export const clearCollectionsSearch = () => ({ type: CLEAR_COLLECTIONS_SEARCH })
 export const filterCollections = (param) => ({ type: FILTER_COLLECTIONS, param: param });
 export const clearCollectionsFilter = (paramKey) => ({ type: CLEAR_COLLECTIONS_FILTER, paramKey: paramKey });
 
+export const cumulusInstanceMetadata = () => wrapRequest(null, get, 'instanceMeta', ADD_CMR);
```

Comment:
Nitpick: can we change this to `getCmrInstanceMetadata`?

Reply:
We could, I left it generic because the API endpoint is `instanceMeta`, and it returns an object with a cmr key right now, but is designed to be extended with any number of other values we might need to ship off to clients in the future.  If you still feel strongly I'll change it.

Reply:
I don't feel strongly about adding `Cmr` to the name, but would you be okay with adding a verb to the front, so `getCumulusIntanceMetadata`?

Reply:
sure